homeassistant/components/supla/__init__.py

```diff
@@ -0,0 +1,165 @@
+"""
+Support for the Supla devices.
+
+For more details about this platform, please refer to the documentation.
+https://home-assistant.io/components/supla/
+"""
```

Comment:
The [link](https://developers.home-assistant.io/docs/en/development_guidelines.html#file-headers) to the documentation is no longer required.

Reply:
Link removed.